src/coreclr/inc/corinfo.h

```diff
@@ -2050,6 +2050,8 @@ class ICorStaticInfo
     // Quick check whether the method is a jit intrinsic. Returns the same value as getMethodAttribs(ftn) & CORINFO_FLG_INTRINSIC, except faster.
     virtual bool isIntrinsic(CORINFO_METHOD_HANDLE ftn) = 0;
 
+    virtual bool notifyMethodInfoUsage(CORINFO_METHOD_HANDLE ftn) = 0;
```

Comment:
It's not obvious to me what this returns -- add some documentation?

Reply:
Sure, will add comments